test/runtime/sungeun/chpl-env-gen.precomp

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
```

Comment:
I like, except `chpl_def_home`, since it makes me feel like I need to maintain that line of code if the directory ever moves.  Is CHPL_HOME ever *not* defined?  If not, I'd just as soon leave off the default value for `getenv`.